tests/bson/bson-document-toPHP-001.phpt

```diff
@@ -0,0 +1,95 @@
+--TEST--
+MongoDB\BSON\Document::toPHP(): __pclass must be both instantiatable and Persistable
```

Comment:
Not reviewing these, as I assume they were just lifted from existing `toPHP()` tests.

Reply:
That assumption would be correct.